### ISSUE TITLE
win32 installer could have a custom icon

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3151]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3151|
|Type|enhancement|
|Created|2008-04-02 20:06:51Z|

That would be awesome.

Attachments:

* [3151.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a6a/a6ab729c76af5396ed12350ddf68178e667372de/c9237a695d2a77258829a87f11a440477812445f.diff) (499 bytes) - added by therve on 2008-04-02 20:07:47Z - 
* [twisted-icon.ico](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a6a/a6ab729c76af5396ed12350ddf68178e667372de/e72792b1ff30a4083940e4e705d8fe0a63c3be7d.ico) (1406 bytes) - added by therve on 2008-04-02 20:08:09Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3151 3151
type__enhancement enhancement
reporter__therve therve
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1207166811000000 1207166811000000
changetime__1207514470000000 1207514470000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix set owner to @therve|
|-|-|

I guess this is fine. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [23219]) Fix various issues with win32 build scripts:
 * Re-include zope.interface
 * Remove useless coil.bat script
 * Add an icon to the installer

Author: therve
Reviewer: radix
Fixes [#3144](https://github.com/twisted/twisted/issues/3144)
Fixes [#3150](https://github.com/twisted/twisted/issues/3150)
Fixes [#3151](https://github.com/twisted/twisted/issues/3151)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
